src/App.js

```diff
@@ -7,7 +7,7 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.js</code> and save to reload.
+          Amjad :(
```

Comment:
did you test this? i think you are based on an old version of the app. try pulling the lasted changes on the main branch and creating a new branch again